nucypher/characters/lawful.py

```diff
@@ -510,6 +510,7 @@ def retrieve(
         alice_verifying_key: PublicKey,  # KeyFrag signer's key
         encrypted_treasure_map: EncryptedTreasureMap,
         publisher_verifying_key: Optional[PublicKey] = None,
+        timeout: int = 10,
```

Comment:
Can we make the `10` as a constant value somewhere?

Reply:
✔️